plugins/github-issues/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@backstage/plugin-github-issues",
+  "version": "0.0.0",
```

Comment:
We will need a changeset too for this PR to make sure that we release this plugin next release

Reply:
Changeset added here: [ffd5e47](https://github.com/backstage/backstage/pull/12875/commits/ffd5e47fb5c6338aa0e6965d687d0028e0baabb5)